front/app/modules/commercial/content_builder/admin/components/RenderNode/index.tsx

```diff
@@ -14,20 +14,37 @@ import { FormattedMessage } from 'utils/cl-intl';
 import messages from '../../messages';
 
 const RenderNode = ({ render }) => {
-  const { isActive } = useEditor((_, query) => ({
+  const {
+    isActive,
+    isDeletable,
+    parentId,
+    actions: { selectNode },
+    query: { node },
+  } = useEditor((_, query) => ({
     isActive: query.getEvent('selected').contains(id),
+    parentId: id && query.node(id).ancestors()[0],
+    isDeletable: id && query.node(id).isDeletable(),
   }));
 
   const { id, name } = useNode((node) => ({
     name: node.data.custom.displayName || node.data.displayName,
   }));
 
-  const nodeNameIsVisible = isActive && id !== ROOT_NODE;
+  useEffect(() => {
```

Comment:
We need this to make sure the columns inside the two column layout are not clickable themselves but we can still drop elements inside of them.